lib/mock/mock-agent.js

```diff
@@ -134,6 +137,92 @@ class MockAgent extends Dispatcher {
   [kGetNetConnect] () {
     return this[kNetConnect]
   }
+
+  [kGetMockAgentClientsByConsumedStatus] () {
+    const mockAgentClients = this[kClients]
+
+    return Array.from(mockAgentClients.entries())
+      .map(([origin, scopeWeakRef]) => {
+        const scope = scopeWeakRef.deref()
+
+        if (scope == null) {
+          throw new Error('scope was null; this should not happen')
+        }
+
```

Comment:
```suggestion
        if (scope == null) {
          throw new Error('scope was null; this should not happen')
        }

```

this is just to please TS